source/pidomus.cc

```diff
@@ -690,6 +690,11 @@ piDoMUS<dim, spacedim, n_components, LAC>::residual(const double t,
   {
     cell->get_dof_indices (data.local_dof_indices);
     this->energy.get_system_residual(cell, scratch, data, data.double_residual);
+    // apply conservative loads
+    this->energy.apply_forcing_terms(cell, scratch, data, data.double_residual);
+
+    if (cell->at_boundary())
+      this->energy.apply_neumann_bcs(cell, scratch, data, data.double_residual);
```

Comment:
no, va benissimo così e' solo il commento sbagliato. I `loads`, per come è scritto qui, sono del tipo **non conservativo**. In particolare un load conservativo è un caso speciale di load **non conservativo**, per cui l'unica cosa sbagliata è il commento.